src/components/lists/_macro.njk

```diff
@@ -59,6 +59,7 @@
                     {%- else -%}
                         <a href="{{ item.url }}" class="list__link {{ item.classes }}" {% if item.index is defined and item.index %}data-qa="list-item-{{ loop.index }}"{% endif %} {% if item.target is defined and item.target %} target="{{ item.target }}"{% endif %} {% if item.name is defined and item.name %} name="{{ item.name }}"{% endif %} {% if item.rel is defined and item.rel %} rel="{{ item.rel }}"{% endif %}>
                             {%- if item.prefix is defined and item.prefix -%}<span class="u-vh">{{- item.prefix -}}</span>{%- endif -%} {{- itemText | safe -}}
+                            {%- if item.target is defined and item.target == "_blank" -%}<span class="u-vh">{{- item.screenReaderMessage | default("this link will open in a new tab") -}}</span>{%- endif -%}
```

Comment:
Probably should create a new ticket up bring the button (`newWindowDescription`) and other instances of vh spans inline with this.  We should settle on a param name for vh spans.  We're using `assistiveTextPrefix` in panel, so this should probably be `assistiveTextSuffix`.

Reply:
New card created https://trello.com/c/6d82zRNz/1085-consistent-name-for-vh-spans-on-links-opening-in-a-new-tab